test/envman.go

```diff
@@ -0,0 +1,45 @@
+package test
```

Comment:
why test is the name of this package and why is it under test/envman.go ?

Reply:
This file will be moved to internal/test/envman.go to keep the reusability and not exporting it.